benchmarks/orc/profile.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+perf record -F 99 -a -g -- $1
```

Comment:
I am happy to add some general helper scripts to profile easier, but this script contains user-specific dirs. Either let's remove the file or make it usable. Prob. easier to remove.